src/ClientContext.php

```diff
@@ -85,7 +85,7 @@ public function forcedTLSVersion()
      *
      * @return void
      */
-    public function setLogger(LoggerInterface $logger)
+    public function setLogger(LoggerInterface $logger): void
```

Comment:
This change break the PHP 5.6 compatibility

Reply:
Good catch, this requires PHP 7.1.0. I actually didn't expect Alma to support PHP 5.6.
I add a note about this in the related issue.